components/ListServices/RegisterForm.jsx

```diff
@@ -242,7 +242,7 @@ const RegisterForm = ({
         </Form.Item>
 
         <Form.Item
-          label="Cost of agent instance bond"
+          label="Cost of agent instance bond (wei)"
```

Comment:
It's wei yes, we could also ask for gwei, for example, but then convert to wei on the JS side. uint96 defines a limiting number of bits of 2^96, since there's not more (and maybe will never be) wei than that.

Reply:
I think this would be a good fix, but could be something we follow up with. This is an improvement, so let's go with this, create a ticket and push forward.